### ISSUE TITLE
Incompatibility with Drush 8 PHAR even after 6.49 upgrade

### ISSUE THREAD

**Issue by evandonovan** [d6lts/drupal#36]
I'm running Drush version 8.1.18 on a site with the latest 6.49 that was upgraded to fix the phar:// vulnerability fixed in https://www.drupal.org/sa-core-2019-002, and had the 7.63 hotfix applied. However, I'm getting a "Drush command terminated abnormally due to an unrecoverable error" message whenever I try to run drush

I don't see any debugging info in the PHP error log.

When i do drush -d, it looks like it is failing during bootstrap_drupal_full().

Could this be related to or https://www.drupal.org/project/drupal/issues/3026560?

However, my Drush appears to work on 7.63, even though 6.49 appears to have the same hotfix code.

I have the drush phar executable at /usr/local/bin/drush, which is a symlink to /usr/local/bin/drush.phar. I've tested running "drush.phar" also.

**Comment by dsnopek** [comment]
It could be related to the similar issues in Drupal 7 & 8. We're waiting for that to all get resolved there before backporting here.

Your best bet is to install Drush via composer rather than the PHAR. What I usually do is something like:

```
mkdir ~/drush
cd ~/drush
composer require drush/drush:8.1.18
export PATH="$HOME/drush/vendor/bin:$PATH"
drush --version
```

I just typed that out without testing so that might not be exactly right, but hopefully gives you the idea :-)

**Comment by evandonovan** [comment]
Thanks, I'll give that a go, and reply back if it resolves the issue.

I think it may be related to the D7 issue, but I also have a 7.63 site on the same server that runs the same instance of Drush with no problems, so I am not sure.

I found some earlier notes I took that suggest it may actually be related to the fact that I have the D6 Securepages module forcing SSL on all my site pages. I'll let you know.

Anyway, I'm hoping the point will be moot soon since I plan to finally migrate this site off D6 in the next few months. It's my last remaining active D6 install.
